Tasks/AzureTestPlanV0/automatedTestInvoker.ts

```diff
@@ -19,14 +21,14 @@ export async function testInvoker(testsToBeExecuted: string[], ciData: ciDiction
         }
 
         switch (testLanguage) {
-            case 'Java-Maven':
-                exitCode = await executeMavenTests(testsToBeExecuted);
+            case 'JavaMaven':
```

Comment:
Add the testing screenshot that without the pom file path provided, we are by default looking at the root of the repository for the pom.xml file.

Reply:
Updated description to add testing screenshots for default project structure